src/transactions/ManageOfferOpFrame.cpp

```diff
@@ -25,6 +25,22 @@ namespace stellar
 
 using namespace std;
 
+static int64_t
```

Comment:
I wonder if those two methods should be merged into one, as getOfferBuyingLiabilities and getOfferSellingLiabilities are mostly called in pairs and it may not be 100% optimal to call exchangeV10WithoutPriceErrorThresholds twice with the same parameters